set a minimum stack size on ppc64 to prevent crashes

Firefox, a user of nspr, attempts to set a stack size of 32k in its
JS watchdog, which the pthreads impl doesn't like and fails.

Coincidentally, nspr here is compiled with debug on, which means it has
assertions turned on; under normal circumstances, those would be disabled,
the stack size set call would fail and everything would fall back on the
default 2M stacksize, but it would still be wrong - therefore, establish
a minimum, just like for aarch64.

--- nspr/pr/include/md/_linux.h
+++ nspr/pr/include/md/_linux.h
@@ -71,7 +71,7 @@
 #define _MD_DEFAULT_STACK_SIZE	65536L
 #define _MD_MMAP_FLAGS          MAP_PRIVATE
 
-#if defined(__aarch64__) || defined(__mips__)
+#if defined(__aarch64__) || defined(__mips__) || defined(__powerpc64__)
 #define _MD_MINIMUM_STACK_SIZE  0x20000
 #endif
 
